output-comparator/src/test/java/org/mobilitydata/gtfsvalidator/outputcomparator/cli/MainTest.java

```diff
@@ -68,7 +69,16 @@ private void writeFile(JsonObject fileData, Path path) throws IOException {
   }
 
   private static String retrieveReportString(Path path) throws IOException {
-    return Files.readString(path);
+    String content = Files.readString(path);
+    Gson gson = new GsonBuilder().setPrettyPrinting().create();
+    JsonObject obj = gson.fromJson(content, JsonObject.class);
+    return gson.toJson(obj);
+  }
+
+  private static String retrieveResource(String name) throws IOException {
```

Comment:
nit[non-blocking]: We can define this method at instance level rather than `static`. The same change can be done to `retrieveReportString` to keep them aligned.

Reply:
Any particular reason to prefer an instance method here as opposed to a static method?

Reply:
This method is not accessing any other static method and can be see as an independent unit of processing.  As it is a private method is not needed to be shared across classes, only internal consumers.

Reply:
Maybe I'm not totally clear on what you are proposing.  When you said "define this method at instance level" rather than static, I assumed you were proposing to remove the `static` qualifier.  Is that correct?  Or were you proposing something different?

Reply:
Sorry for the misunderstanding, remove the static qualifier.